Doc/library/ctypes.rst

```diff
@@ -309,10 +309,10 @@ bytes objects are immutable)::
    >>> s = "Hello, World"
    >>> c_s = c_wchar_p(s)
    >>> print(c_s)
-   c_wchar_p('Hello, World')
+   c_wchar_p(140018365411392)
    >>> c_s.value = "Hi, there"
    >>> print(c_s)
```

Comment:
@berkerpeksag I changed to `print(c_s, c_s.value)`, since the content says that "c_wchar_p and c_void_p changes the memory location they point to...", I think observe the memory location changed in `c_s` is useful, too.